### DISCUSSION TITLE
Antd 5.0.2 is slow rendering components

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#39081] [Q&A]
Hi guys, I'm new in ant design. I've created a dashboard following the 5.0.2 components, but all seems are very slow.
Meaning render time is slow.
Example try a dropdown here: https://ant.design/components/dropdown
Example of "faster option": https://rsuitejs.com/components/dropdown

Anyone can help?

**Comment by zombieJ** [comment]
You can set customize token to change the motion duration. Default is `0.3s`

**Reply by vitalijalbu** [reply]
where to find them? i'd like to custom all time transitions

**Reply by zombieJ** [reply]
Use ConfigProvider:
https://codesandbox.io/s/basic-usage-deprecated-syntactic-sugar-antd-5-0-2-forked-m3mz85?file=/demo.tsx

**Comment by abenhamdine** [comment]
@vitalijalbu what do you mean by "slow rendering" ? What is slow ?

**Reply by vitalijalbu** [reply]
try to see the links i attached, you can see that it takes more time to display the dropdown

**Reply by abenhamdine** [reply]
I actually tried the links, but "rendering" could mean many things.
It looks like you are refering to the time to display the dropdown panel, not the time to render the component.

**Comment by afc163** [comment]
Same as https://github.com/ant-design/ant-design/issues/38686
